README.md

```diff
@@ -27,247 +28,229 @@ which functions as the data backbone for prominent hedge funds, banks, and finan
+The following list contains documentation links for installation instructions and more:
 
-Developers interested in tinkering with and modifying source code should build from the source code. For detailed instructions on how to do this, see [Build and launch Deephaven](https://deephaven.io/core/docs/how-to-guides/launch-build).
+- Python
+  - [Run from Docker](https://deephaven.io/core/docs/tutorials/quickstart/)
+  - [pip-installed](https://deephaven.io/core/docs/tutorials/quickstart-pip/)
+- Groovy
+  - [Run from Docker](https://deephaven.io/core/groovy/docs/tutorials/quickstart/)
+- [Python client](https://pypi.org/project/pydeephaven/)
+- [Java client](https://deephaven.io/core/docs/how-to-guides/java-client/)
+- [JS client](https://deephaven.io/core/docs/reference/js-api/documentation/)
+- [Go client](https://pkg.go.dev/github.com/deephaven/deephaven-core/go)
+- [R client](https://github.com/deephaven/deephaven-core/blob/main/R/rdeephaven/README.md)
 
-If you are not sure which of the two is right for you, use the pre-built images.
+## Install and run Deephaven
 
-### Required Dependencies
+The Deephaven server can be installed and instantiated [from Docker](#from-docker), [from Python](#from-python), or [from source code](#built-from-source).
 
-Running Deephaven requires a few software packages.
+### From Docker
 
-| Package        | Version                       | OS           |
-| -------------- | ----------------------------- | ------------ |
-| docker         | ^20.10.8                      | All          |
-| docker-compose | ^1.29.0                       | All          |
-| Windows        | 10 (OS build 20262 or higher) | Only Windows |
-| WSL            | 2                             | Only Windows |
+This is the easiest way to get started with Deephaven. For full instructions, see our [quickstart for Docker](https://deephaven.io/core/docs/tutorials/quickstart/). The table below shows installation dependencies.
 
-You can check if these packages are installed and functioning by running:
-```
-docker version
-docker-compose version
-docker run hello-world
-```
-
-> :warning: **On Windows, all commands must be run inside a WSL 2 terminal.**
+| Dependency     | Version  | OS      | Required/Recommended |
+| -------------- | -------- | --------| -------------------- |
+| Docker         | ^20.10.8 | All     | Required             |
+| Docker compose | ^2       | All     | Recommended          |
+| Windows        | 10+      | Windows | Required             |
+| WSL            | ^2       | Windows | Required             |
 
-If any dependencies are missing or unsupported versions are installed, see [Launch Deephaven from pre-built images](https://deephaven.io/core/docs/tutorials/quickstart#prerequisites) for installation instructions.
+The quickest way to install and run Deephaven from Docker is with a single Docker command.
```

Comment:
The command should come immediately after, IMO, or else move this line to just under the next heading.